packages/transport/src/core/Transport.ts

```diff
@@ -36,6 +38,8 @@ export interface IConfigOverwrite {
     platformClientId: string;
     platformClientSecret: string;
     supportedIdentityVersion: number;
+    supportedMinBackboneVersion?: number;
+    supportedMaxBackboneVersion?: number;
```

Comment:
this should not be overridable

Reply:
I put the overrides in for testing - changing the limits is necessary for the test if there is no mock